### PR TITLE
modify the command line examples

### PR DESCRIPTION
**Pull request by Stevent-fei**
### Describe what this PR does / why we need it
1. Update readme
2. modify image name

### CODE REVIEW

File: utils/test/file/Clusterfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ spec:
     bindPort: 6443
   nodeRegistration:
     criSocket: /var/run/dockershim.sock
-  kubernetesVersion: v1.19.8
+  kubernetesVersion: v1.22.15
```
Comment (kakaZhou719) [inline utils/test/file/Clusterfile:26]: we don’t need to replace this for the test image

File: README.md [hunk 2]
```diff
@@ -53,52 +53,86 @@ wget https://github.com/sealerio/sealer/releases/download/v0.9.0/sealer-v0.9.0-l
...
+    matchLabels:
+      run: my-nginx
+  template:
+    metadata:
+      labels:
+        run: my-nginx
+    spec:
+      containers:
+        - name: my-nginx
+          image: nginx
```
Comment (kakaZhou719) [inline README.md:87]: use full path : docker.io/library/nginx:latest

File: cmd/sealer/cmd/image/build.go [hunk 3]
```diff
@@ -52,17 +52,17 @@ a brand new ClusterImage.`
...
 build without cache:
-  sealer build -f Kubefile -t my-kubernetes:1.19.8 --no-cache .
+  sealer build -f Kubefile -t docker.io/sealerio/kubernetes:v1.22.15 --no-cache .
 build with args:
-  sealer build -f Kubefile -t my-kubernetes:1.19.8 --build-arg MY_ARG=abc,PASSWORD=Sealer123 .
+  sealer build -f Kubefile -t docker.io/sealerio/kubernetes:v1.22.15 --build-arg MY_ARG=abc,PASSWORD=Sealer123 .
 build with image type:
-  sealer build -f Kubefile -t my-kubernetes:1.19.8 --type=app-installer .
-  sealer build -f Kubefile -t my-kubernetes:1.19.8 --type=kube-installer(default) .
+  sealer build -f Kubefile -t docker.io/sealerio/kubernetes:v1.22.15 --type=app-installer .
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/build.go:61]: sealer build -f Kubefile -t myapp:v1 --type=app-installer

File: cmd/sealer/cmd/image/build.go [hunk 4]
```diff
@@ -52,17 +52,17 @@ a brand new ClusterImage.`
 
 var exampleNewBuildCmd = `the current path is the context path, default build type is lite and use build cache
 build:
-  sealer build -f Kubefile -t my-kubernetes:1.19.8 .
+  sealer build -f Kubefile -t docker.io/sealerio/kubernetes:v1.22.15
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/build.go:55]: this is build image name , i think no need to modify it.

File: README.md (hunk 2 above)
Comment (Stevent-fei) [reply README.md:87]: > 

@kakaZhou719  This is defined in yaml and does not need to be modified

File: cmd/sealer/cmd/cluster/run.go [hunk 5]
```diff
@@ -60,6 +60,17 @@ func NewRunCmd() *cobra.Command {
 		Long:    longNewRunCmdDescription,
 		Example: exampleForRunCmd,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO: remove this now, maybe we can support it later
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/run.go:63]: mark

delete unused code

**Review by Starnop — APPROVED** [review]
LGTM

**Pull request merged**